molecule/mysql/converge.yml

```diff
@@ -0,0 +1,52 @@
+---
+- name: Converge
+  hosts: all
+
+  vars:
+    inventory_dir: "{{ lookup('env', 'MOLECULE_SCENARIO_DIRECTORY') }}/../../environments/vm"
+    mariadb_cluster_name: molecule
+    mariadb_cluster_password: secret
+    mariadb_root_password: secret
+    mariadb_backup_password: secret
+    galera_bootstrap_node: openconext-centos7-mysql
+
+  pre_tasks:
+    - name: Read vars from secrets file
+      include_vars: "../../environments/vm/secrets/vm.yml"
+    - name: Read vars from group_vars file
+      include_vars: "../../environments/vm/group_vars/vm.yml"
+
+    - name: "Get ip current host"
+      shell: grep $(hostname) /etc/hosts | awk '{print $1}' | sort | tail -n 1
+      register: current_host_ip
+      changed_when: False
```

Comment:
Is it not possible to get the host IP address using gather_facts with molecule?

Reply:
I had issues before in using the gather_facts for getting the last ip (basically the network that Molecule is creating), so since then I use these tasks with every Molecule test that I have created.